### PR TITLE
[AIRFLOW-3724] Fix the broken refresh button on Graph View in RBAC UI

### PR DESCRIPTION
**Pull request by kaxil**
Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-3724

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:
Pressing on the refresh button in Graph View results in an error `error: INTERNAL SERVER ERROR` as shown in the screenshot below.

image

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:
We don't test UI stuff currently
### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [x] In case of new functionality, my PR adds documentation that describes how to use it.
  - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.
  - All the public functions and the classes in the PR contain docstrings that explain what it does

### Code Quality

- [x] Passes `flake8`

### CODE REVIEW

**Comment by kaxil** [comment]
cc @verdan

File: airflow/www/templates/airflow/graph.html [hunk 1]
```diff
@@ -112,8 +112,8 @@
     // Below variables are being used in dag.js
     var tasks = {{ tasks|safe }};
     var task_instances = {{ task_instances|safe }};
-    var getTaskInstanceURL = '{{ url_for("Airflow.task_instances", dag_id='+
-      dag.dag_id + ', execution_date=' + execution_date + ') }}';
+    var getTaskInstanceURL = "{{ url_for('Airflow.task_instances',
+      execution_date=execution_date)}}&dag_id={{dag.dag_id}}";
```
Comment (verdan) [inline airflow/www/templates/airflow/graph.html]: the way we are not using the `dag_id` as a parameter in url_for function and only using execution date is kinda confusing. Either we should use both as parameters or skip both. JS does not provide a proper way to use jinja template variables, so I'd suggest to use both variables outside the `url_for` function. 
Something like:

```
    var getTaskInstanceURL = "{{ url_for('Airflow.task_instances') }}" +
                            "?dag_id=" + encodeURIComponent(dag_id) +
                            "&execution_date=" + encodeURIComponent(execution_date);
```

**Review by verdan — CHANGES_REQUESTED** [review]

File: airflow/www/templates/airflow/graph.html (hunk 1 above)
Comment (kaxil) [reply airflow/www/templates/airflow/graph.html]: Updated as per your suggestion

**Comment by abitrolly** [comment]
> We don't test UI stuff currently

Hmm..

**Pull request merged**